Assets/LeapMotion/Scripts/DetectionUtilities/Detector.cs

```diff
@@ -0,0 +1,53 @@
+﻿using UnityEngine;
+using UnityEngine.Events;
+using System.Collections;
+using Leap;
+
+namespace Leap.Unity {
+
+  public class Detector : MonoBehaviour {
+    public bool IsActive{ get{ return _isActive;} private set { _isActive = value;}}
```

Comment:
Having a private setter in this case isn't needed since there already is a way to set the variable privately.  Having a setter also might open the door for potential bugs if additional code is later added to the setter unintentionally.